src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustCorporateDto.java

```diff
@@ -26,6 +26,10 @@ public class TrustCorporateDto {
 
     public static final String IS_ON_REGISTER_IN_COUNTRY_FORMED_IN = "is_on_register_in_country_formed_in";
 
+    public static final String CORPORATE_BODY_STILL_INVOLVED_IN_TRUST = "is_corporate_body_still_involved_in_trust";
```

Comment:
For consistency within this class I would rename this constant to `IS_CORPORATE_BODY_STILL_INVOLVED_IN_TRUST` to match the same pattern as the constant defined above - `IS_ON_REGISTER_IN_COUNTRY_FORMED_IN`.

Reply:
Suffixed these fields.

Reply:
Thanks for suffixing with `_FIELD` but please prefix with `IS_`.